modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/service/MotechSchedulerListener.java

```diff
@@ -0,0 +1,68 @@
+package org.motechproject.scheduler.service;
+
+import org.joda.time.DateTime;
+import org.motechproject.event.MotechEvent;
+import org.motechproject.event.listener.annotations.MotechListener;
+import org.motechproject.scheduler.contract.JobId;
+import org.motechproject.scheduler.contract.RepeatingJobId;
+import org.motechproject.scheduler.contract.RepeatingSchedulableJob;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Map;
+
+/**
+ * This service is used to schedule/unschedule jobs through the motech event.
+ */
+@Service
```

Comment:
Look more like a component. Morever, I don't think this class should be exposed looking at its methods, yet it's placed in the service package. Can you move this class to a better place?